src/renderers/dom/shared/ReactDOMComponentTree.js

```diff
@@ -95,7 +92,8 @@ function precacheChildNodes(inst, node) {
   }
   var children = inst._renderedChildren;
   var childNode = node.firstChild;
-  outer: for (var name in children) {
+  outer:
```

Comment:
This seems to indicate to me that labels are available anywhere on their own but they're not actually valid in front of anything - although almost.